File: mix-fruit-juice/mix-fruit-juice.js
function mixFruit (arr) {
var reg = [ "banana", "orange", "apple", "lemon", "grapes"];
var spec = ["avocado", "strawberry" , "mango"];
var s = arr.reduce((s,i) => reg.concat(spec).includes(i.toLowerCase())?(reg.includes(i.toLowerCase())?s+5:s+7):s+9,0)
return Math.round(s/arr.length)
}

File: mix-fruit-juice/mix-fruit-juice.py
def mix_fruit(arr):
    dictio = {"banana":5, "orange":5, "apple":5, "lemon":5, "grapes":5,"avocado":7,"strawberry":7,"mango":7}     
    return round(sum([dictio.get(i.lower(),9) for i in arr])/len(arr))